_guides/how-we-do-development.md

```diff
@@ -200,6 +215,8 @@ when you are finished and it's safe to merge in. But don't be precious about it.
 If someone else needs to merge your code in, they should ask you about it if
 they can, but shouldn't feel prevented from doing so.
 
+When merging code to main, we prefer to use a merge commit, rather than merging with a rebase, as having a merge commit as part of the project history makes it clear when a change has actually been merged into the codebase.
```

Comment:
```suggestion
When merging code into a long-lived branch, we prefer to use a merge commit, rather than merging with a rebase, as having a merge commit as part of the project history makes it clear when a change has actually been merged into the codebase.
```